dev/com.ibm.ws.jpa.tests.jpa_32_fat/test-applications/jakartadata/src/io/openliberty/jpa/data/tests/web/JakartaDataRecreateServlet.java

```diff
@@ -168,6 +168,52 @@ public void testOLGH28913() throws Exception {
 
         assertEquals(character.getHexadecimal(), result);
     }
+    
+    @Test
+    @Ignore("Reference issue:https://github.com/OpenLiberty/open-liberty/issues/29459")
+    public void testOLGH29459() throws Exception {
+        // Sample segment ID and points
+        int x1 = 0, y1 = 0, x2 = 120, y2 = 209;
+        
+        // Create a new segment using the 'of' factory method
+        Segment segment = Segment.of(x1, y1, x2, y2);
+        
+        // Begin the transaction
+        tx.begin();
+
+        try {
+            // Persist the segment using EntityManager
+            em.persist(segment);
```

Comment:
This persist operation will not attempt to persist the entity until `tx.commit()` is called.
Therefore, you are not testing the same behavior as the original issue.
(see next comment for a suggestion on how to do this correctly)